docs/decision_records/api/API-008-multi-state-store-api-design.md

```diff
@@ -0,0 +1,30 @@
+# API-008: Multi State store API design
+
+## Status
+Accepted
+
+## Context
+This decision record is to support multiple state stores support in Dapr. We agreed on the decision to introduce the breaking change in API
+to support multi state store with no backward compatibility.
+
+With this change , the state API allows the app to target a specific state store by store-name, for example:
+
+v1.0/state/storeA/

```

Comment:
We should indicate which field in the yaml the store-name comes from.  Having 2 yaml example yamls with 2 corresponding urls will help clarify.